src/fontra/client/web-components/glyphs-search.js

```diff
@@ -68,8 +69,8 @@ export class GlyphsSearch extends UnlitElement {
       { key: "glyphName", title: "glyph name", width: "10em", isIdentifierKey: true },
       {
         key: "unicode",
-        width: "5em",
-        get: (item) => makeUPlusStringFromCodePoint(item.unicodes[0]),
+        width: "fit-content",
```

Comment:
This is perhaps fine for now, but relates to #756. This is currently only working correctly because it is the last column...